src/block_cache/block_chunkcache.c

```diff
@@ -566,7 +574,7 @@ __wt_chunkcache_get(WT_SESSION_IMPL *session, WT_BLOCK *block, uint32_t objectid
              * would be spin-waiting for this chunk to become valid. The current thread will mark
              * the chunk as valid, and any waiters will unblock and proceed reading it.
              */
-            (void)__wt_atomic_addv32(&chunk->valid, 1);
+            WT_PUBLISH(chunk->valid, true);
```

Comment:
Can we add a comment here that the `PUBLISH` and `ORDERED_READ`s are required since populating the chunk isn't protected by the bucket_lock?

Reply:
Good call - done in 8334956.